api/core/v1alpha1/bucketclaim_helpers.go

```diff
@@ -71,6 +71,9 @@ func BucketBindToMark(logsetMeta metav1.ObjectMeta) string {
 }
 
 func AddBucketFinalizer(ctx context.Context, c client.Client, lsMeta metav1.ObjectMeta, finalizer string) error {
+	if lsMeta.Namespace == "" || lsMeta.Name == "" {
+		return nil
+	}
```

Comment:
Is this possible?

Reply:
probably impossible, may be I am too sensitive